source/iterators.tex

```diff
@@ -1377,7 +1369,40 @@
 \returns \tcode{addressof(operator*())}.
 \end{itemdescr}
 
-\rSec4[reverse.iter.op++]{\tcode{operator++}}
+\indexlibrarymember{operator[]}{reverse_iterator}%
+\begin{itemdecl}
+constexpr @\unspec@ operator[](difference_type n) const;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\returns
+\tcode{current[-n-1]}.
+\end{itemdescr}
+
+\indexlibrarymember{operator+}{reverse_iterator}%
+\begin{itemdecl}
+constexpr reverse_iterator operator+(difference_type n) const;
```

Comment:
I find it a bit strange to call this an "observer", since it does not of itself provide any mechanism to actually observe the state of the iterator. (It seems a little odd to call `operator*`, `operator->`, and `operator[]` observers, rather than something like "element access", too).

I might be tempted to instead lump `operator++`, `operator--`, `operator+`, `operator+=`, `operator-`, `operator-=` under something like "`reverse_iterator` navigation", put these three under "`reverse_iterator` element access", and keep `base` as a single-item subclause (which I think makes sense, as it's the one and only part of the `reverse_iterator` interface that is not part of some Iterator concept).

Reply:
Sounds reasonable.